### PR TITLE
CG quiesce post upgrade validation test

### PR DESCRIPTION
**Pull request by sumabai**
# Description

Adding CG quiesce upgrade automation to existing post upgrade validation script as new module.

Jira task: https://issues.redhat.com/browse/RHCEPHQE-14102 

Script : tests/cephfs/cephfs_upgrade/cephfs_post_upgrade_validation.py

Polarion TC link : https://polarion.engineering.redhat.com/polarion/redirect/project/CEPH/workitem?id=CEPH-83581470 

Log: http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-1LQIBA

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/cephfs/cephfs_upgrade/cephfs_post_upgrade_validation.py [hunk 1]
```diff
@@ -741,11 +758,245 @@ def mds_config_test(mds_req_params):
...
+                )
+
+    if cg_test_io_status.value == 1:
+        log.error(
+            f"CG IO test exits with failure during quiesce lifecycle with await on qs_set-{qs_id_val}"
+        )
+        test_fail = 1
+
+    if test_fail == 1:
+        log.error("FAIL: CG quiesce lifecycle on set of existing and new subvolumes")
```
Comment (neha-gangadhar) [inline tests/cephfs/cephfs_upgrade/cephfs_post_upgrade_validation.py:995]: Can we differentiate if the quiesce failed on existing pre-upgrade ones or new subvolumes post upgrade?

**Review by neha-gangadhar — COMMENTED** [review]
lgtm

File: tests/cephfs/cephfs_upgrade/cephfs_post_upgrade_validation.py (hunk 1 above)
Comment (sumabai) [reply tests/cephfs/cephfs_upgrade/cephfs_post_upgrade_validation.py:995]: Yes Neha,
i am reporting quiesce test failure based on IO validation tool, and in this tool, i am reporting subvolume name if IO validation fails, so we will know which subvolume caused failure, as subvolume name has pre_upgrade in its name for existing volumes

**Review by AmarnatReddy — APPROVED** [review]
LGTM

**Pull request merged**